Dockerfile

```diff
@@ -56,6 +56,8 @@ COPY ansible/requirements.yml /ansible-requirements.yml
 
 COPY package*.json ./
 
+RUN pip3 install -U pip
```

Comment:
It’s better to combine it to the same command with `&&` to avoid creating additional fs layers.